### ISSUE TITLE
Image normalization on my own dataset. 

### ISSUE THREAD

**Issue by VikasRajashekar** [open-mmlab/mmdetection#6900]
Hello, 

I have a doubt about image normalization. 
I have my own dataset in COCO format for instance segmentation. 

I tried DCN. https://github.com/open-mmlab/mmdetection/blob/master/configs/dcn/cascade_mask_rcnn_r101_fpn_dconv_c3-c5_1x_coco.py

So the config comes with its own img_norm_cfg.
But my dataset has following values:
```
img_norm_cfg = dict(
    mean=[128, 128, 128], std=[11.578, 11.578, 11.578], to_rgb=True)
```

So when I am fine-tuning on my own dataset do I need to use the default img_norm_config or the values calculated from my own dataset? 

Will it have an impact on the results.?

**Comment by hhaAndroid** [comment]
@VikasRajashekar If it is fintune mode, you do not need to modify this field.

**Comment by VikasRajashekar** [comment]
@hhaAndroid in case I **don't want** to use the provided pre-train checkpoint using load_from. 
In that case, can I use img_norm_cfg of my own dataset?

**Comment by hhaAndroid** [comment]
@VikasRajashekar You just need to ensure that the norm value is consistent with the pre-training.
